src/funlist.c

```diff
@@ -464,6 +464,60 @@ FUNCTION(fun_graball)
     free_ansi_string(as);
 }
 
+/* ARGSUSED */
+FUNCTION(fun_arrow)
+{
+  /* iteratively evaluates a list of attributes, with a single argument 
+   * and optional arguments. It passes the result of each result to the 
+   * next attribute as %0, similar to fold, but carries all optional argument
+   * info down each level. Repeat until there are no attributes left in the 
+   * list. Fails if there is no argument.
+   */
+
+  PE_REGS *pe_regs;
+  int funccount, per;
+  char result[BUFFER_LEN];
+  char *list[MAX_SORTSIZE] = {NULL};
```

Comment:
This provides an explicit initializer for the first element of the array—and then the rest happen to be zero-initialized as well. Probably just write `= {}`?

Though since this is an out-param for list2arr_ansi below, and it seems much more common to completely omit the initializer in such code in Pennmush.

Reply:
I was following the pattern found in `fold()`. Will make this change.

Reply:
```
funlist.c:480:30: warning: ISO C forbids empty initializer braces [-Wpedantic]
   char *list[MAX_SORTSIZE] = {};
```

Turns out, that is why.

Reply:
Initializer removed as of next commit.